sql/core/src/main/scala/org/apache/spark/sql/jdbc/OracleDialect.scala

```diff
@@ -230,3 +224,11 @@ private case object OracleDialect extends JdbcDialect {
 
   override def supportsOffset: Boolean = true
 }
+
+private[jdbc] object OracleDialect {
```

Comment:
Ur, actually, `case object` has more features (serializable, hashCode, toString) than `object`. Do we need to shrink features like this?

Reply:
This is only a place for containing some const ints, which represent some variants from Oracle JDBC extentions. Thus, `object` is enough

Reply:
Opps, the errors occured
```
- simple scan with LIMIT *** FAILED *** (65 milliseconds)
[info]   org.apache.spark.SparkException: Task not serializable
```